kube_log_watcher/agents/scalyr.py

```diff
@@ -31,6 +31,15 @@ def __init__(self, cluster_id: str, load_template):
                 'Scalyr watcher agent initialization failed. {} config path does not exist.'.format(
                     self.config_path))
 
+        if not os.path.exists(self.dest_path):
```

Comment:
It is a kubernetes volume, so it must be there in order to function properly. This path is actually shared with Scalyr-agent.